docs/tutorials/airflow-trigger-airbyte-job.md

```diff
@@ -0,0 +1,117 @@
+
+First of all, make sure you have Docker and Docker Compose installed. 
+
+### **Start Airbyte**
+For this tutorial you need have a connection created. If you are starting with Airbyte and want to test the integration, we suggest to you follow the [Getting Started tutorial](../getting-started.md) and use that connection.
+
+The recommendation is to set up the **sync frequency** of your Connection to **manual** because Airflow will become responsible to trigger this process.
+
+
+### **Start Airflow**
+
+To setup Airflow we recommend you to follow this [Quick start Airflow](https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html) if you don't have any instances yet.
+
+
+## 2. Create a DAG in Airflow to trigger your Airbyte job
+
+You can access Airflow at [http://localhost:8080/](http://localhost:8080/).
+
+### Create Airbyte connection in Airflow
+After Airflow starts, access the connections page to create our connection with Airbyte.
+
+![](../.gitbook/assets/airflow_create_connection.png)
+
+Insert data about our connection. We are going to use the Airbyte API to execute our actions. For this reason we will create an HTTP connection.
+
+![](../.gitbook/assets/airflow_edit_connection.png)
+
+After inserting the information click on Save.
+
+### Getting information about your Airbyte connection job
+You can access Airbyte at `localhost:8000`. 
+![](../.gitbook/assets/airflow_airbyte_connection.png)
+That code in the url is the `connectionId` used by Airflow to trigger the job. In our example: `1e3b5a72-7bfd-4808-a13c-204505490110`
+
+### Create a simple DAG in Airflow to run a sync job
+Create a file inside the `/dags` directory called `dag_airbyte_example.py`
+```python
+from airflow import DAG
+from airflow.utils.dates import days_ago
+from airflow.providers.airbyte.operator import AirbyteTriggerSyncOperator
+
+with DAG(dag_id='trigger_airbyte_job_example',
+         default_args={'owner': 'airflow'},
+         schedule_interval='@daily',
+         start_date=days_ago(1)
+    ) as dag:
+
+    money_to_json = AirbyteTriggerSyncOperator(
+        task_id='airbyte_money_json_example',
+        airbyte_conn_id='airbyte_conn_example',
+        connection_id='1e3b5a72-7bfd-4808-a13c-204505490110',
+        asynchronous=False,
+        timeout=360,
+        wait_seconds=5
+    )
+```
+About the parameters used in the Operator:
+- `airbyte_conn_id`: the HTTP connection configured above.
+- `connection_id`: The uuid code for the connection job. You can get this in the url path from Airbyte UI.
+- `asynchronous`: this is specific to Airflow DAG execution. Default is `false`. This is useful for submitting long running jobs and         waiting on them asynchronously using the **AirbyteJobSensor**. If you have a very busy Airflow instance this could be handful because after submiting the job you use the sensor to check, and a sensor doesn't occupy a worker slot in Airflow. Later we show how to configure a DAG using this configuration.
```

Comment:
```suggestion
- `asynchronous`: Determines how the Airbyte Operator runs. When true, Airflow will monitor the Airbyte Job via the **AirbyteJobSensor**. Since sensors do not occupy an Airflow worker slot, this is helps reduce Airflow load. Default is `false.
```